src/assets/js/diy.js

```diff
@@ -1,16 +1,16 @@
 (() => {
     var tracker
 
-    const components = [
+    const components = [ //Prices and names for component categories
```

Comment:
If you add comments please consider using JSDoc standard.
```suggestion
    /**
     * @typedef {Object} Choice
     * @property {string} name Name of the part.
     * @property {(function(number):number|function():number|number)} amount  Calculated amount element of part based on provided set parts count.
     * @property {number} cost Calculated or hardcoded cost element of part.
     * @property {(function(number):number|function():number|number)} costAll Calculated total cost element of part based on provided set parts count.
     * @property {string} links HTML string with purchase links/info.
     */

    /**
     * @typedef {Object} Component
     * @property {string} name Name of the component category.
     * @property {Choice[]} choices
     * @property {boolean} [hideFor5Set]
     */

    /** @type {Component[]} */
    const components = [
```